docs/docs/setup/flutter-setup.md

```diff
@@ -1,6 +1,6 @@
 # Flutter Setup
 
-After completing the [Rust setup](rust-setup.md) and setting up either [iOS setup](ios-setup.md) or [Android setup](android-setup.md), you're ready to create a cross-platform project using [Flutter](https://flutter.dev/). <br/>
+Using ["Getting Started - 3. Mopro build"](/docs/getting-started.md#3-build-bindings) guide, you can generate the "MoproAndroidBindings" and "MoproIOSBindings" for the ios and android platforms in your project folder, you're ready to create a cross-platform project using [Flutter](https://flutter.dev/). <br/>
```

Comment:
The official usage is `iOS`. We should keep this aligned in the paragraph.
```suggestion
Using ["Getting Started - 3. Mopro build"](/docs/getting-started.md#3-build-bindings) guide, you can generate the "MoproAndroidBindings" and "MoproiOSBindings" for the iOS and android platforms in your project folder, you're ready to create a cross-platform project using [Flutter](https://flutter.dev/). <br/>
```

Reply:
Thanks for picking up this.